### DISCUSSION TITLE
Doris 创建 hive 外表时报错

### DISCUSSION THREAD

**Discussion by CodeCooker17** [apache/doris#20424] [A - General / Q&A]
CREATE TABLE `hive_v2` (
  `tag_id`  varchar(128) ,
  `tag_value` varchar(128) ,
  `case_id` int
) ENGINE=HIVE
COMMENT "HIVE"
PROPERTIES (
'hive.metastore.uris' = '***',
'database' = 'dm_dmp',
'table' = 'auto_user_is_aiqicha_company_tag_v2',
'dfs.nameservices'='***',
'dfs.ha.namenodes.hacluster'='nn1,nn2',
'dfs.namenode.rpc-address.hacluster.n1'='***',
'dfs.namenode.rpc-address.hacluster.n2'='***',
'dfs.client.failover.proxy.provider.hacluster'='org.apache.hadoop.hdfs.server.namenode.ha.ConfiguredFailoverProxyProvider'
);

通过这种方式创建 hive 外表时报错：Execution failed: Error Failed to execute sql: java.sql.SQLException: (conn=279) Unexpected exception: failed to check if table auto_user_is_aiqicha_company_tag_v2 in db dm_dmp exists. reason: shade.doris.hive.org.apache.thrift.TApplicationException: Invalid method name: 'get_table_req'

hive 版本2.1.2

创建 catalog 是可以指定 hive 版本的，但是创建 hive 外表时不能指定 hive 版本，有什么好的方式解决这个问题吗

**Comment by yagagagaga** [comment]
Now it is recommended to use a catalog instead of External Table. External Table has been marked as deprecated. If your Doris does not support Hive Catalog, you can upgrade to the latest version [1.2.4.1(Stable)](https://doris.apache.org/download).

**Reply by CodeCooker17** [reply]
spark load must deploy hive external table. 
I solved this problem, and I will contribute the code later.  https://github.com/apache/doris/issues/20588
